boards/common/esp8266/include/periph_conf.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_esp8266 ESP8266 Common
+ * @{
+ *
+ * @file
+ * @brief       Configurations of MCU periphery that are common for all ESP8266 boards
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "c_types.h"
+
+// include board.h to make board specific configuration visible in any case
```

Comment:
Please change to C style comments (`/* * */`)

Reply:
Is there any reason for that? I couldn't find any hint in your coding style about comments and ANSI C99 standard allows // as comment markers.

Reply:
You are right. C99 standard allows C++ style comments and our Coding Convention didn't explicitly say C-style should be used. Nevertheless, the reason why we do this is the fact we try to follow Linux Kernel coding convention (which surprisingly they also didn't say anything about C++ comments, but it's reflected in their code).

We already added the [entry](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#comments) in our Coding Convention. Thanks for pointing it out!